1.5/examples/index.md

```diff
@@ -78,4 +78,4 @@ features of Apache Accumulo.
                        Also shows user permissions.
 
 
-[1]: {{ site.baseurl }}/1.5/accumulo_user_manual#_user_administration
+[1]: {{ site.baseurl }}/1.5/user-manual#_user_administration
```

Comment:
This doesn't necessarily account for all the external user section links, in the mailing list, on stack-overflow, etc.